website/docs/src/content/index.mdx

```diff
@@ -6,7 +6,7 @@ sidebarTitle: Overview
 
 # GenSX Overview
 
-GenSX is a simple typescript framework for building complex LLM applications. It's a workflow engine designed for building agents, chatbot APIs, and more using common patterns like RAG and reflection.
+GenSX is a simple TypeScript framework for building complex LLM applications. It’s a workflow engine designed for building agents, chatbot APIs, and more using common patterns like RAG and reflection.
```

Comment:
We should add a second sentence that mentions the cloud here now that it's discussed in this doc. Essentially, we want to convey that GenSX is a) a typescript framework for building lllm workflows/agents and b) there is a gensx cloud that offers tracing/deployemnt/storage